subprojects/tooling-api/tooling-api.gradle.kts

```diff
@@ -98,11 +98,9 @@ gradlebuildJava {
 apply(from = "buildship.gradle")
 
 tasks.sourceJar {
```

Comment:
This should now be `sourcesJar` (with **s**) which is added by `withSourcesJar()`.

`sourceJar` is defined here: https://github.com/gradle/gradle/blob/648979e3252cb63108cc6881ace5f97d56bb51ef/buildSrc/subprojects/plugins/src/main/kotlin/gradlebuild/publish-public-libraries.gradle.kts#L36-L39

That definition should be removed and all places where `sourceJar` is used should be updated to use `sourcesJar`.

Reply:
Agreed. I noticed this, but did not want to touch with this change. Will clean it up then.